src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -236,20 +238,20 @@ function prepareMCQQuestions() {
 
                 event.stopImmediatePropagation();
             });
-        }
-    }
+        });
+    });
 }
 
 function prepareContribQuestions() {
     var contribQuestionNums = getQuestionTypeNumbers('CONTRIB');
 
-    for (var i = 0; i < contribQuestionNums.length; i++) {
+    $.each(contribQuestionNums, function(i) {
         var qnNum = contribQuestionNums[i];
 
         // Get number of options for the specified question number of contribution question type
         var optionNums = $('[name^="responsetext-' + qnNum + '-"]').length;
 
-        for (var k = 0; k < optionNums; k++) {
+        $.each(optionNums, function(k) {
```

Comment:
Fixed this one, almost every test passes now, except one (see above comment). xD
Thanks for the catch!